### ISSUE TITLE
Fix Flaky test ReadTxnWriteAutoScaleWithFailoverTest.readTxnWriteAutoScaleWithFailoverTest

### ISSUE THREAD

**Issue by shshashwat** [pravega/pravega#6462]
**Describe the bug**
system test ReadTxnWriteAutoScaleWithFailoverTest.readTxnWriteAutoScaleWithFailoverTest fails with following exception

```
java.lang.AssertionError: readTxnWriteAutoScaleWithFailoverTest test failed due to Error with message null
	at io.pravega.test.system.framework.services.kubernetes.K8SequentialExecutor.lambda$startTestExecution$4(K8SequentialExecutor.java:93)
```

**Comment by SrishT** [comment]
This test has been consistently failing in the last multiple system test runs, with the following exception
```
2022-02-16 06:35:41,103 740187 [Time-limited test] INFO i.p.t.system.AbstractReadWriteTest - Wait for readers execution to complete
2022-02-16 06:35:41,104 740188 [Time-limited test] INFO i.p.t.system.AbstractReadWriteTest - Unable to read events:
java.lang.RuntimeException: [java.io](http://java.io/).EOFException
at io.pravega.client.stream.impl.JavaSerializer.deserialize(JavaSerializer.java:63)
at io.pravega.client.stream.impl.JavaSerializer.deserialize(JavaSerializer.java:32)
at io.pravega.client.stream.impl.EventStreamReaderImpl.readNextEventInternal(EventStreamReaderImpl.java:184)
at io.pravega.client.stream.impl.EventStreamReaderImpl.readNextEvent(EventStreamReaderImpl.java:132)

2022-02-16 06:35:41,105 740189 [Time-limited test] ERROR i.p.t.system.SingleJUnitTestRunner - Test readTxnWriteAutoScaleWithFailoverTest failed with exception.
java.lang.AssertionError: Unable to read events. Test failure
at org.junit.Assert.fail(Assert.java:88)
at io.pravega.test.system.AbstractReadWriteTest.stopReaders(AbstractReadWriteTest.java:397)
```

**Comment by RaulGracia** [comment]
The serialization issue was fixed with this PR #6645. Please, create a new issue for this test if it fails due to some other reason.
